configs/permissions/vulnerability.json

```diff
@@ -1,3 +1,9 @@
 {
-    "*": ["*"]
+    "*": ["*", "read", "write"],
+    "vulnerability_results": ["read", "write"],
+    "cve.business_risk": ["write"],
```

Comment:
are "read" permissions not necessary for this operation?

Reply:
`read` permission not necessary here because it covered in `vulnerability_results`